src/components/RequestCard.tsx

```diff
@@ -16,21 +13,30 @@ export const RequestCard = ({
   description,
   comments,
 }: ProductRequest) => {
-  let userComments = comments?.length === undefined ? 0 : comments?.length;
+  const userComments = comments?.length === undefined ? 0 : comments?.length;
```

Comment:
How about this logic?
```
const numberOfUserComments: number = comments?.length || 0;
```

Note the variable name change and the TS type. This reads a little nicer